src/three-components/TextureUtils.ts

```diff
@@ -260,6 +254,115 @@ export default class TextureUtils extends EventDispatcher {
     }
   }
 
+  gaussianBlur(
+      cubeTarget: WebGLRenderTargetCube,
+      standardDeviationRadians: number = 0.04) {
+    const blurScene = new Scene();
+
+    const geometry = new BoxBufferGeometry();
+    geometry.removeAttribute('uv');
+
+    const cubeResolution = cubeTarget.width;
+    const standardDeviations = 3;
+    const n = Math.ceil(
+        standardDeviations * standardDeviationRadians * cubeResolution * 2 /
+        Math.PI);
+    const norm = standardDeviations / ((n - 1) * Math.sqrt(2 * Math.PI));
+    let weights = [];
+    for (let i = 0; i < n; ++i) {
+      const x = standardDeviations * i / (n - 1);
+      weights.push(norm * Math.exp(-x * x / 2));
+    }
+
+    const blurMaterial = new ShaderMaterial({
```

Comment:
If we make the `blurMaterial` and `blurCamera` class members, we could probably re-use them and then dispose them when `TextureUtils` dispose is called.

Reply:
Actually we can't reuse blurMaterial because if you call the function with a different standard deviation, you'll create a different shader. Likewise if you invoke it on a different size texture, you'll have to create a differently sized camera.

Reply:
👍